frappe/desk/page/setup_wizard/install_fixtures.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+
+import frappe
+
+from frappe import _
+
+default_genders = ["Male", "Female", "Other"]
+
+default_salutations = ["Dr.", "Mr.", "Ms.", "Mx.", "Sir", "Dame", "Esq.", "Lady", "Lord", "Mrs.", "Sire", "Madam", "Miss.", "Master", "Proff.", "Captain", "Mistress", "Gentleman", "President", "Principal"]
```

Comment:
Lets not add the period `.` also Professor is `Prof` not `Proff`... `Mistress` ?? 😜